### DISCUSSION TITLE
markdown related tools

### DISCUSSION THREAD

**Discussion by rockonedege** [CorentinTh/it-tools#836] [Ideas]
Would love to have tools converting to/from markdown, say csv,json.

**Comment by sarah-schuh** [comment]
I second this. would be much appreciated!

**Comment by sharevb** [comment]
Hi @rockonedege and @sarah-schuh, implemented in many PR : https://github.com/CorentinTh/it-tools/pulls?q=is%3Apr+is%3Aopen+markdown

**Comment by sharevb** [comment]
Hi, if you arr interested in up to date version of it-tools, I made kind of a fork here : https://github.com/sharevb/it-tools (https://sharevb-it-tools.vercel.app/ and docker images https://github.com/sharevb/it-tools/pkgs/container/it-tools)
